dataactvalidator/validation_handlers/validator.py

```diff
@@ -421,17 +424,39 @@ def evaluateMultiFieldRule(rule, record, interfaces, fileType):
         ruleType = rule.multi_field_rule_type.name.upper()
         if(ruleType == "CAR_MATCH"):
             # Look for an entry in car table that matches all fields
-            fieldsToCheck = Validator.cleanSplit(rule.rule_text_1)
-            tasFields = Validator.cleanSplit(rule.rule_text_2)
+            fieldsToCheck = cls.cleanSplit(rule.rule_text_1)
+            tasFields = cls.cleanSplit(rule.rule_text_2)
             if(len(fieldsToCheck) != len(tasFields)):
                 raise ResponseException("Number of fields to check does not match number of fields checked against",StatusCode.CLIENT_ERROR,ValueError)
-            return Validator.validateTAS(fieldsToCheck, tasFields, record, interfaces, fileType)
+            return cls.validateTAS(fieldsToCheck, tasFields, record, interfaces, fileType)
         elif(ruleType == "SUM_TO_VALUE"):
-            return Validator.validateSum(rule.rule_text_1, rule.rule_text_2, record)
+            return cls.validateSum(rule.rule_text_1, rule.rule_text_2, record)
+        elif(ruleType =="SUM_FIELDS"):
+            value = record[FieldCleaner.cleanName(rule.rule_text_1)]
+            if value is None or value =="":
+                value = 0
+            return cls.validateSum(value, rule.rule_text_2, record)
         elif(ruleType == "REQUIRE_ONE_OF_SET"):
-            return Validator.requireOne(record,rule.rule_text_1.split(','),interfaces)
+            return cls.requireOne(record,rule.rule_text_1.split(','),interfaces)
+        elif(ruleType == "NOT"):
+            # Negate the rule specified
+            conditionalRule = interfaces.validationDb.getMultiFieldRuleByLabel(rule.rule_text_1)
+            return not cls.evaluateMultiFieldRule(conditionalRule,record,interfaces,fileType)
+        elif(ruleType == "REQUIRED_CONDITIONAL"):
+            # Field in rule_text_1 is required if rule in rule_text_2 is satisfied
+            conditionalRule = interfaces.validationDb.getMultiFieldRuleByLabel(rule.rule_text_2)
+            if cls.evaluateMultiFieldRule(conditionalRule,record,interfaces,fileType):
+                # Return True if field populated
+                field = FieldCleaner.cleanName(rule.rule_text_1)
+                if field in record and cls.isFieldPopulated(record[field]):
+                    return True
+                else:
```

Comment:
Also if the behavior in the if clause ever changed to not immediately return it would not break the behavior of returning False if the field is not present.